LayoutTests/platform/ios-simulator/TestExpectations

```diff
@@ -72,6 +72,10 @@ imported/w3c/web-platform-tests/html/canvas/offscreen/wide-gamut-canvas/2d.color
 imported/w3c/web-platform-tests/webcodecs/videoFrame-createImageBitmap.any.worker.html [ Failure ]
 imported/w3c/web-platform-tests/webcodecs/videoFrame-drawImage.any.worker.html [ Failure ]
 
+# Missing required low latency H264 support.
+webkit.org/b/261118 imported/w3c/web-platform-tests/webcodecs/temporal-svc-encoding.https.any.worker.html?h264 [ Pass Failure ]
```

Comment:
if it's missing feature, why would it pass ?

Reply:
Changed to failure only.